pkg/cvo/cvo_scenarios_test.go

```diff
@@ -975,13 +1115,16 @@ func TestCVO_StartupAndSyncPreconditionFailing(t *testing.T) {
 			},
 			VersionHash: "DL-FFQ2Uem8=",
 			History: []configv1.UpdateHistory{
+				//{State: configv1.PartialUpdate, Image: "image/image:1", Version: "1.0.0-abc", StartedTime: defaultStartedTime},
```

Comment:
nit, drop this line?

Reply:
Addressed in https://github.com/openshift/cluster-version-operator/pull/745